docs/scanpipe-tutorial-1.rst

```diff
@@ -59,13 +59,13 @@ Step-by-step
   pipeline run::
 
     $ scanpipe show-pipeline --project staticbox
-    "[SUCCESS] scanpipe/pipelines/docker.py"
+    "[SUCCESS] docker"
 
 - Get the results of the pipeline run as a JSON file using the ``output`` command::
 
-    $ scanpipe output --project staticbox results.json
+    $ scanpipe output --project staticbox --format json
 
-- Open the ``results.json`` in your preferred viewer.
+- Open the ``output/results-<timestamp>.json`` in your preferred viewer.
```

Comment:
IMHO the UX would be better if we instead require the output file to be named? Or at least it should be possible to name the file. I find sometimes frustrating to have files created or display and I cannot control their name? To discuss!

Reply:
This requires its own ticket.